### DISCUSSION TITLE
Welcome to homeassistant-innova Discussions!

### DISCUSSION THREAD

**Discussion by danielrivard** [danielrivard/homeassistant-innova#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ggraziotti** [comment]
Can I use this integration with 3in1 model heat pump ?
This integration is useful also for convectors ?

**Reply by danielrivard** [reply]
I don't think it would work as is, those models may have different functionality that are not covered by this integration.

They may also use a different API. I cannot really tell at this point, since I don't own any of these other models, and the API is not really publicly documented.

**Comment by Cametes** [comment]
This integration is great! Have been using it for a few months now, and I'm very glad it exists!

**Comment by Oliver171068** [comment]
Good evening, I have carried out the installation according to https://github.com/danielrivard/homeassistant-innova. When entering the IP address I get the error message "failed to connect". what am I doing wrong?

**Reply by danielrivard** [reply]
if you try to open this url (replacing with the IP of your innova) http://192.168.XX.XX/api/v/1/status what do you see?

**Reply by Oliver171068** [reply]
No matter which IP I enter from the innova, I receive the information. The website is not accessible

**Reply by danielrivard** [reply]
Are you sure your Innova is connected to your network?

**Reply by Oliver171068** [reply]
I connect the air conditioning to WiFi. This is also how it appears to me in the Innova APP. My router shows me that it is connected via LAN, which is not true.
![IMG_7789](https://github.com/user-attachments/assets/de36f848-4bd6-4a8a-a936-251865e3be21)
![IMG_7788](https://github.com/user-attachments/assets/57623df9-87d3-449f-bc90-161d92f37cbe)

**Reply by danielrivard** [reply]
Just to be sure, which version of the Innova application are you using?

Because recent Innova units are not compatible anymore with my integration, that might be the reason. 
If that is the case, unfortunately, they completely changed the protocol used to communicate with the App.

**Reply by Oliver171068** [reply]
it is version 2.6.0 (2)
